root/cdstub/BrowseCDStubs.js

```diff
@@ -27,8 +27,8 @@ const BrowseCDStubs = ({
   cdStubs,
   pager,
 }: Props): React$Element<typeof Layout> => (
-  <Layout fullWidth title={l('Top CD Stubs')}>
```

Comment:
This needs to be changed also in `t/lib/t/MusicBrainz/Server/Controller/CDStub/Browse.pm`